test/test_config.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:test/test.dart';
+import 'package:draw/src/drawConfigContext.dart';
+
+int main() {
+  test('Tests for default section of local file', () {
```

Comment:
We also need to run some tests with:

- Custom fields
- Empty fields
- Invalid configuration files
- Partially populated configurations to check defaults
- Accessing fields in non-default/non-global sections in the configuration file